internal/utils/sliceutils/sliceutils.go

```diff
@@ -0,0 +1,13 @@
+package sliceutils
```

Comment:
I'm not wild about the depth of the package tree here: `internal/utils/sliceutils/sliceutils.go`

Can we put this `sliceutils.go` file right in the `utils` package?

Reply:
I did that based on feedback I got a while ago from uhhh... Dan or Nick, idr, about worries that a single `utils` package would just balloon -- but i don't feel strongly.

Reply:
I agree that's a good point, could we make `sliceutils` package at the top level?

Reply:
sure, will do that